lib/widgets/content.dart

```diff
@@ -663,12 +665,14 @@ class MessageImage extends StatelessWidget {
           src: resolvedSrcUrl,
           thumbnailUrl: resolvedThumbnailUrl,
           originalWidth: node.originalWidth,
-          originalHeight: node.originalHeight));
+          originalHeight: node.originalHeight,
+          pageContext: pageContext));
```

Comment:
While testing this, I tried passing `context` to `pageContext` (and below), and it appears to work. Let's try this so that we don't need to rely on having a PageRoot.